### PR TITLE
initial migration of raffle planck

### PR DESCRIPTION
**Pull request by raffled**
Contains my keymap for the planck.

### CODE REVIEW

File: keyboards/planck/keymaps/raffle/config.h [hunk 1]
```diff
@@ -0,0 +1,43 @@
+#ifndef CONFIG_USER_H
+#define CONFIG_USER_H
+
+#include "config_common.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/raffle/config.h]: This line isn't needed.  Could you remove it?

File: keyboards/planck/keymaps/raffle/config.h [hunk 2]
```diff
@@ -0,0 +1,43 @@
+#ifndef CONFIG_USER_H
```
Comment (drashna) [inline keyboards/planck/keymaps/raffle/config.h]: Could you replace these two lines with just `#pragma once`, and remove the `#endif` at the end?

File: keyboards/planck/keymaps/raffle/keymap.c [hunk 3]
```diff
@@ -0,0 +1,312 @@
...
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "planck.h"
```
Comment (drashna) [inline keyboards/planck/keymaps/raffle/keymap.c]: Could you replace these two includes with `#include QMK_KEYBOARD_H` ?

File: keyboards/planck/keymaps/raffle/keymap.c [hunk 4]
```diff
@@ -0,0 +1,312 @@
...
+      break;
+    case R_CMLN:
+      if (record->event.pressed) {
+	SEND_STRING("################################################################################");
+      }
+      return false;
+      break;
+    case SHRG:
+      if (record->event.pressed) {
+	SEND_STRING("¯\\_(ツ)_/¯");
```
Comment (drashna) [inline keyboards/planck/keymaps/raffle/keymap.c]: This probably won't work, actually.  

If it does, how? :)

**Comment by raffled** [comment]
config is cleaned up.  Good call on the shrug, totally forgot that I never did get that working.  Thanks!

File: keyboards/planck/keymaps/raffle/keymap.c (hunk 3 above)
Comment (drashna) [reply keyboards/planck/keymaps/raffle/keymap.c]: You missed this, I think.

**Comment by drashna** [comment]
As for the shrug, this may work: https://github.com/qmk/qmk_firmware/blob/master/users/drashna/drashna.c#L364-L387

**Comment by drashna** [comment]
Thanks!

**Pull request merged**